oldwinapi/windows.nim

```diff
@@ -9136,8 +9140,8 @@ type
     rclBounds*: RECTL
     nPolys*: DWORD
     cptl*: DWORD
-    aPolyCounts*: array[0..0, DWORD]
-    aptl*: array[0..0, POINTL]
+    aPolyCounts*: UncheckedArray[DWORD]
+    aptl*: UncheckedArray[POINTL]

```

Comment:
This cannot be correct either. Only the last field in an object can be an UncheckedArray.

Reply:
`aptl` and `abTypes` are just pointers to "flexible arrays", so I think this is fine.
https://msdn.microsoft.com/en-us/library/windows/desktop/dd162558(v=vs.85).aspx